playbooks/roles/content_host/tasks/main.yml

```diff
@@ -0,0 +1,9 @@
+- name: 'Install subscription-manager'
+  yum: name=subscription-manager
+
+- name: 'Install bootstrap rpm'
+  command: 'rpm -Uvh http://{{ katello_server }}/pub/katello-ca-consumer-latest.noarch.rpm'
```

Comment:
This RPM requires sub-man, so if you use yum to install it you don't need to install sub-man in line 1.